woocommerce/woocommerce
# Additional changes for the core profiler plugins page

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #38436  .

This PR addresses changes listed in the issue, except for the Jetpack redirection and `InstallAndActivatePlugins` refactor. I'll handle Jetpack redirection in a separate issue, as it requires a bit of context switching.  @rjchow is working on refactoring `InstallAndActivatePlugins` in https://github.com/woocommerce/woocommerce/pull/38577. For the reference, my previous attempt at refactoring is https://github.com/woocommerce/woocommerce/commit/94e548cec115ef4c4a9ed2739b76c542882c4d56



### How to test the changes in this Pull Request:

For the label and description, please refer to Figma design Y5pUYSJPsGEud1vknUZhi8-fi-300-14366 or use the screenshot: 
![image](https://github.com/woocommerce/woocommerce/assets/9312929/589c657a-4b3c-4f7d-9d96-e0767e594018)


Make sure to turn off `Show Suggestions` from WooCommerce -> Settings -> Advanced -> WooCommerce.com to show the local version of the suggestions.

1. Go to the Plugins page
2. Confirm the plugins have the correct label and description.
3. Confirm the plugins have a `learn more link
4. Select some plugins and click `Continue`
5. Confirm the plugins get installed in order of [install_priority](https://github.com/woocommerce/woocommerce/compare/update/38436-additional-changes-for-plugins-page?expand=1#diff-f508723b8f4a18b29a46152942aa4b58d21a5ddf06866e1238a734bd458f4eb6)
6. Confirm the change requests from the label and description

For now, I've set `install_priority` value by plugin size. 

| Plugin  |  Priority | Size|
| ------------- | ------------- | ------------- |
| Tiktok for Business  | 1  |  54 KB |
| Pinterest for WooCommerce  | 2  | 985 KB |
| WooCommerce Shipping| 3 | 1.6 MB |
| WooCommerce Tax | 4| 1.6 MB|
| WooCommerce Payments| 5| 5 MB|
| Google Listing and Ads|6|7.2MB|
|Mailpoet|7|7.4MB|
|Jetpack|8|25.7MB|
